src/user/formats/hal.ts

```diff
@@ -94,7 +97,8 @@ export function item(user: User, privileges: PrivilegeMap, hasControl: boolean,
     createdAt: user.createdAt.toISOString(),
     modifiedAt: user.modifiedAt.toISOString(),
     type: user.type,
-    privileges
+    privileges,
+    userInfo,
```

Comment:
Lets just make this part of the user response body instead of embedded in an object, so:

```suggestion
    ...userInfo,
```

Reply:
This `userInfo` also includes a `createdAt` and `modifiedAt`, which a spread operator would overwite on the user record... is that ok?

Reply:
Yeah this is why later in this review I suggested to just remove them. It's not needed for the client to have several date fields here, they're going to be the same usually anyway.